tests/test_local_bindings.py

```diff
@@ -155,7 +155,7 @@ def test_internal_locals_differ():
     assert m.local_cpp_types_addr() != cm.local_cpp_types_addr()
 
 
-@pytest.mark.xfail("env.PYPY")
+@pytest.mark.xfail("env.PYPY and sys.pypy_version_info < (7, 3, 2)")
```

Comment:
"Expect this to fail on PyPy < 7.3.2"

Didn't we decide to drop older versions already?

Reply:
7.3.2 is not out, yes. And it still fails on 7.3.1 :-)

Reply:
If 7.3.2 is out before this gets merged, yes, let's remove it

Reply:
(unrelated change, btw, but something I bumped into, when testing locally)

Reply:
I probably wouldn't drop it right away; approximately supporting 7.3.0+ would probably be ideal.

Reply:
Yes, that's probably best.